dart/constraint/BoxedLcpConstraintSolver.cpp

```diff
@@ -169,7 +169,7 @@ void BoxedLcpConstraintSolver::solveConstrainedGroup(ConstrainedGroup& group)
   mFIndex.setConstant(n, -1); // set findex to -1
 
   // Compute offset indices
-  mOffset.resize(n);
+  mOffset.setZero(numConstraints);
```

Comment:
FYI, this change is not necessarily part of the fix, but I'm backporting from an upstream [PR](https://github.com/dartsim/dart/pull/1459) since it's related.